catalogs/sources/gtfs/schedule/us-california-wasco-1782.json

```
@@ -0,0 +1,16 @@
+{
+    "mdb_source_id": 1782,
+    "data_type": "gtfs",
+    "provider": "City of Wasco",
+    "features": ["flex","flex-v2"],
+    "status": "development",
+    "location": {
+        "country_code": "US",
+        "subdivision_name": "California",
+        "municipality": "Wasco"
+        }
+    },
```

The JSON is invalid.
```suggestion
    },
```